### ISSUE TITLE
Create CONTRIBUTING.md file

### ISSUE THREAD

**Issue by mhmdanas** [TeamNewPipe/NewPipeExtractor#704]
This should probably copy most of https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md save for NewPipe-specific parts.

Some other things to note:
- No Kotlin is allowed in the codebase

**Comment by opusforlife2** [comment]
Can we just copy NP's file over?

**Comment by mhmdanas** [comment]
I guess that would work, but then we would need to sync the two versions together, right?

Another option is to share the same contributing guidelines, but note when guidelines differ between the two projects. Yet another option (that I do not prefer) is to link to the NewPipe guidelines in NPE, and then state the differences.

**Comment by opusforlife2** [comment]
Hmmm. Syncing seems like it would be an effort to set up. I would go with the least complicated option of linking the URL and noting the differences.

**Comment by opusforlife2** [comment]
What say?

**Comment by mhmdanas** [comment]
Sharing the contributing guidelines between both projects is a better idea in my opinion, as the idea of linking to the NP guidelines in the NPE guidelines might cause the NPE guidelines to get outdated.

**Comment by opusforlife2** [comment]
My understanding is that some sort of GH Action would need to be set up to keep them in sync.

**Comment by mhmdanas** [comment]
No, my idea is to link to the NP guidelines in the NPE guidelines, but state the differences in the earlier.

**Comment by opusforlife2** [comment]
Ah. Makes complete sense.

**Comment by arunkd13** [comment]
What happened to this?

Do we have a list of differences? Or should we just create the initial version referring to the NewPipe version?
